data/mapcss-mapping.csv

```diff
@@ -1131,20 +1131,20 @@ deprecated|deprecated;1132;x
 deprecated|deprecated;1133;x
 deprecated|deprecated;1134;x
 deprecated|deprecated;1135;x
-deprecated|deprecated;1136;x
-deprecated|deprecated;1137;x
-deprecated|deprecated;1138;x
-deprecated|deprecated;1139;x
-deprecated|deprecated;1140;x
-deprecated|deprecated;1141;x
-deprecated|deprecated;1142;x
-deprecated|deprecated;1143;x
-deprecated|deprecated;1144;x
-deprecated|deprecated;1145;x
-deprecated|deprecated;1146;x
-deprecated|deprecated;1147;x
-deprecated|deprecated;1148;x
-deprecated|deprecated;1149;x
+sport|padel;1136;
+leisure|pitch|padel;[leisure=pitch][sport=padel];x;name;int_name;1137;leisure|pitch
```

Comment:
leisure|pitch|xxx are not needed here. Keep only sport|xxx types. Remove them.

Existing types like leisure|pitch|tennis are present here for the _backward compatibility_ with old map data.